chapter-01-polynomes.tex

```diff
@@ -433,21 +433,21 @@ \section{Divisibilité et racines}
 \end{displaymath}
 tel que $\deg(r(x)) < \deg(g(x))$. On  obtient alors
 \begin{displaymath}
-  f(x) = \left(q(x) + \frac{a_n}{ b_m } x^{n-m} \right) g(x) + r(x). 
+  f(x) = \left(q(x) + \frac{a_n}{ b_m } x^{n-m} \right) g(x) + r(x) = q_0(x)g(x) + r(x), \;\text{où} q_0(x) = q(x) + \frac{a_n}{ b_m } x^{n-m}. 
 \end{displaymath}
 %
-Supposons maintenant qu'il existe deux autres polynômes $q'(x)$ et $r'(x)$ tels que 
+Supposons maintenant qu'il existe deux autres polynômes $q'(x) \neq q_0(x)$ et $r'(x) \neq r(x)$ tels que 
 \begin{displaymath}
     f(x) = q'(x) g(x) + r'(x) 
   \end{displaymath}
   et $\deg(r') < \deg(g)$. 
 Alors 
 \begin{displaymath}
-   r(x) - r'(x) = (q(x) - q'(x)) ⋅ g(x). 
+   r(x) - r'(x) = (q'(x) - q_0(x)) ⋅ g(x). 
```

Comment:
Same issue as above.

Reply:
Just to clarify: I only changed $q(x)$ to $q_0(x)$ ($q'(x)$ is the same as yours, I just changed the order), since, as above, I believe $q(x)$ has already been used, and I think when you say $q(x)$ at that point you actually mean $q(x) + \frac{a_n}{ b_m } x^{n-m}$.

Reply:
See above